### DISCUSSION TITLE
Asset path wrong parsed (probably)

### DISCUSSION THREAD

**Discussion by r4v3n6101** [bevyengine/bevy#5704] [Q&A] [answered]
Hello. Recently I've found a strange logic of loading assets [here](https://github.com/bevyengine/bevy/blob/dc3b4b6c850898c922dff9fd6d312823e07096f1/crates/bevy_asset/src/path.rs#L157)
If we use something with 2 "#" in asset_path then we never get full label because only 2 parts of `label` taken (`asset.zip#123#abc` => `asset.zip` as path and  `123` as label). Probably there should be used the `splitn` method, am I right? Should we ever use 2 "#" in paths?

**Comment by heavyrain266** [comment]
I think, it was mostly designed as label for glTF scenes, like `asset.load("cube.glb#Scene0")`, thats also how we use it in external fbx plugin as `asset.load("cube.fbx#Scene0")`.

**Reply by r4v3n6101** [reply]
And what if `Scene0` named as `Scene#0`? It will be unexpectedly treated as `Scene`, not `Scene#0`

**Comment by r4v3n6101** [comment, marked as answer]
So, it was fixed in here: https://github.com/bevyengine/bevy/issues/5707. It was an issue, not discussion and I was mistaken about open it here.
